backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -94,21 +93,9 @@ public void setDateTestedBackdate(Date date) {
   // - this method is temporary
   // Eventually, this method will be deprecated in favor of getResultSet() and getResultForDisease()
   public TestResult getTestResult() {
-    Hibernate.initialize(this.results);
-    if (this.results != null) {
-      Comparator<Result> resultDateComparator = Comparator.comparing(Result::getUpdatedAt);
-      Optional<Result> resultObject = this.results.stream().max(resultDateComparator);
-      if (resultObject.isPresent()) {
-        return resultObject.get().getTestResult();
-      }
-    }
-    return super.getResult();
-  }
-
-  @JsonIgnore
-  public Set<Result> getResultSet() {
```

Comment:
(non-blocking/question for my own knowledge) When would we use this method as opposed to `getResults()`?

Reply:
This method is only used to resolve the field [`TestOrder.TestResults`](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/java/gov/cdc/usds/simplereport/api/queue/ApiTestOrderDataResolver.java#L103-L107). I just put an `@Deprecated` on this method so no one uses this by accident in the future

Reply:
Given that the `results` field is also deprecated in the `TestOrder` on `main.graphqls`, I think it should be safe to remove the reference from `ApiTestOrder` as well. Then we can remove this `getTestResult` method entirely, instead of just deprecating it. (I just tested locally, and graphQL doesn't blow up when you remove the deprecated `result` out of `ApiTestOrder` - I think at that point, it's the responsibility of the client not to call it). 
I'd personally feel a bit better about removing this method entirely, because the possibility of returning null makes me nervous 😅

Reply:
Removed it in this commit ebc34c9859a314bc210370ca938d55b30d474451